src/board_controller/brainalive/brainalive.cpp

```diff
@@ -1,24 +1,20 @@
-#include <string>
-
 #include "brainalive.h"
 #include "custom_cast.h"
 #include "get_dll_dir.h"
 #include "timestamp.h"
+#include <string.h>
 
-// common constants
-#define BRAINALIVE_PACKET_SIZE 224
 
 // info about services and chars
 #define START_BYTE 0x0A
 #define STOP_BYTE 0x0D
+#define BRAINALIVE_HANDSHAKING_PACKET_SIZE 6
+
+int software_gain, hardware_gain, reffrence_volatage;
```

Comment:
as far as I understand it these values are device speciffic and we determine their values in BF during handshacking. It makes a lot of sense to convert them to class fields, set to 0 in constructor, read, and use in notifications